hq/markdown/snyk.md

```diff
@@ -51,7 +49,7 @@ on the PATH.  As a result it is necessary to pre-pend the sbt location onto the
 
 ```
   "devDependencies": {
-    "snyk": "1.69.3"
+    "snyk": "1.68.0"
```

Comment:
🤔 Is there a reason this is changing?

Reply:
Recommendations from Snyk because the 1.69 releases remove minor bugs.  At least one of those bugs was involved in the difficulties in setting up Snyk for SHQ.

We should use the versions we then suggest others use, for dogfood reasons.

Reply:
> We should use the versions we then suggest others use

Agree 💯 

...However, SHQ specifies `"snyk": "1.69.3"` in the package.json so this should be the one we are using?

Also, earlier in this file ([line 40](https://github.com/guardian/security-hq/pull/80/files/27d812f38bda4a7ec8a4de4a9d97b0b648e40e20#diff-ca96dc326cfaa20524b64a8facbcf9b6R40)) version `"1.69.3"` is also specified

<img width="356" alt="picture 195" src="https://user-images.githubusercontent.com/8607683/35741346-7dd2463e-082f-11e8-986e-0021a41081e2.png">

Is there a reason we mention two different versions?